GiniCapture/Classes/Core/GiniConfiguration.swift

```diff
@@ -623,6 +623,16 @@ import UIKit
      */
     @objc public var shouldShowDragAndDropTutorial = true
     
+    /**
+     Sets if the custom help menu items should be added on the help menu screen
+     */
+    @objc public var hasCustomMenuItems = false
+
+    /**
+     Set an array of additional custom help menu items . Those items will be presented as table view cells on the help menu screen. By selecting the cell the user will be redirected to the page, which represented by viewController provided by customer during the  `HelpMenuViewController.MenuItem` initialization.
+    */
+    @objc public var customMenuItems: [HelpMenuViewController.MenuItem] = []
```

Comment:
I have a "radical" suggestion: we could keep the Item enum and add a `case custom(String, UIViewController)` to it.

Then here we can use `HelpMenuViewController.Item`:
```suggestion
    @objc public var customMenuItems: [HelpMenuViewController.Item] = []
```

Continued below ->